source/reports/ml_reports/MotifSeedRecovery.py

```diff
@@ -211,6 +211,7 @@ def _plot(self, plotting_data, output_name):
                 "coefficients": self._y_axis_title
             }, template='plotly_white',
                          color_discrete_sequence=px.colors.diverging.Tealrose)
+            # figure.update_layout(title={"text":self.title, "x":0.5, "font": {"size":14}})
```

Comment:
should this be used or not?

Reply:
it is intentionally commented out, but I used it when re-plotting some stuff, so it would be useful for me to keep it as a comment..